server/src/validations/Auth.validation.js

```diff
@@ -1,6 +1,7 @@
 import isEmpty from '../helpers/isEmpty';
 import isValidEmail from '../helpers/validateEmail';
 import containsAlphabets from '../helpers/containsAlphabets';
+
 /**
  * @exports
  *@class AuthValidation
```

Comment:
Please leave a space between `*` and `@class` as in 
```suggestion
 * @class AuthValidation
```